File: Model-View-Controller/Go/model/Student.go
package model

//--------------Model----------
type Student struct {
	name   string
	rollNo string
}

func (s *Student) Student(name, rollNo string) {
	(*s).name = name
	(*s).rollNo = rollNo
}

func (s *Student) GetRollNo() string {
	return (*s).rollNo
}
func (s *Student) GetName() string {
	return (*s).name
}
func (s *Student) SetName(name string) {
	(*s).name = name
}
func (s *Student) SetRollNo(rollNo string) {
	(*s).rollNo = rollNo
}

File: Model-View-Controller/C++/Model/Student.cpp
#include <bits/stdc++.h>

using namespace std;

// Student Model
class Student {
    private:
        string rollNo;
        string name;
    public:
        Student() {
            rollNo = "";
            name = "";
        }
        Student(string name, string rollNo) {
            this->rollNo = rollNo;
            this->name = name;
        }
        string getRollNo() {
            return rollNo;
        }
        string getName() {
            return name;
        }
        void setName(string name) {
            this->name = name;
        }
        void setRollNo(string rollNo) {
            this->rollNo = rollNo;
        }
};
